README.md

```diff
@@ -1000,6 +993,14 @@ __AWESOME__ apps counter: __119_ 🎉
 - [x] [GitHub](https://github.com/bitwarden/mobile)
 - [x] [Official page](https://bitwarden.com/)
 
+### Keepass2Android
+<img alt="Keepass2Android" height="64" src="https://github.com/PhilippC/keepass2android/blob/master/graphics/launcher_icon/Logo-green.svg">
+
+- [x] [Google Play](https://play.google.com/store/apps/details?id=keepass2android.keepass2android)
+- [x] [IzzyOnDroid](https://android.izzysoft.de/repo/apk/keepass2android.keepass2android_nonet) offline-only version
```

Comment:
Changes this line to
`- [x] [IzzyOnDroid (Offline-only)](https://android.izzysoft.de/repo/apk/keepass2android.keepass2android_nonet)`